lib/Doctrine/Adapter/Statement/Interface.php

```diff
@@ -29,6 +29,9 @@
  * @link        www.doctrine-project.org
  * @since       1.0
  * @version     $Revision: 7490 $
+ *
+ * Cannot be defined on an interface
+ * @property string $queryString
```

Comment:
Out of curiosity, why is this being added?

Reply:
Because of this line here: https://github.com/diablomedia/doctrine1/blob/master/lib/Doctrine/Connection/Statement.php#L81

`$_stmt` there is identified as "Doctrine_Adapter_Statement_Interface|PDOStatement". The only two classes in Doctrine that extend this interface do have that property defined:

https://github.com/diablomedia/doctrine1/blob/master/lib/Doctrine/Adapter/Statement/Mock.php#L47
https://github.com/diablomedia/doctrine1/blob/master/lib/Doctrine/Adapter/Statement/Oracle.php#L40

But this interface does not (not possible to define properties on an interface).

This could be fixed in a number of different ways, but this was the easiest (and doesn't touch code). :)

Reply:
btw, if I remove that property, this is the error output by phpstan:

```
------ --------------------------------------------------------------------------------------------------
  Line   lib/Doctrine/Connection/Statement.php
 ------ --------------------------------------------------------------------------------------------------
  84     Access to an undefined property Doctrine_Adapter_Statement_Interface|PDOStatement::$queryString.
 ------ --------------------------------------------------------------------------------------------------
```